src/azure-cli/azure/cli/command_modules/security/_params.py

```diff
@@ -340,3 +343,8 @@ def load_arguments(self, _):
             c.argument(
                 'group_name',
                 arg_type=adaptive_application_controls_group_name)
+
+    for scope in ['setting']:
+        with self.argument_context('security {}'.format(scope)) as c:
+            c.argument('setting_name', help='The name of the setting', arg_type=get_enum_type(Enum69))
```

Comment:
You can use `options_list=['--name', '-n']` for setting_name argument to support users use `--name` in command

```suggestion
            c.argument('setting_name', options_list=['--name', '-n'], help='The name of the setting', arg_type=get_enum_type(Enum69))
```